### PR TITLE
[DOC] Remove Spark from our naming of Tools [skip ci]

### LINKED ISSUE #5516: [DOC] Remove Spark from our naming of Qualification Tool
## Report incorrect documentation
Looks like we missed removing "Spark" from couple of our github pages for the Qualification Tool. See https://nvidia.github.io/spark-rapids/docs/spark-qualification-tool.html

 Originally tried to clean up in https://github.com/NVIDIA/spark-rapids/issues/4398 but missed some.  "Spark Qualification tool"

### PR DESCRIPTION
**Pull request by amahussein**
fixes #5516

Signed-off-by: Ahmed Hussein (amahussein) <a@ahussein.me>

This is a followup on #4398 to remove the terminology Spark Profiling tool and Spark Qualification from public documentation.

**Important:**
One thing to take into consideration is that the changes rename two files `docs/spark-qualification-tool.md` and `docs/spark-profiling-tool.md` to `docs/qualification-tool.md` and `docs/profiling-tool.md`.
This may break external links/URLS referring to `docs/spark-qualification-tool.html` and `docs/spark-profiling-tool.html`

**Changes to the code in .md (no need to change scala code):**

- Change all occurrences of `Spark qualification tool` and `Spark profiling tool` to `Qualification tool` and `Profiling tool`
- Chang upper-case of `q(Q)ualification` and `p(P)rofiling`
- rename two files, and fix urls referencing those two files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,17 +52,17 @@ You may need to disable RMM caching when exporting data to an ML library as that
...
+The Qualification tool is used to look at a set of applications to determine if the RAPIDS Accelerator for Apache Spark
 might be a good fit for those applications.
 
-The profiling tool generates information which can be used for debugging and profiling applications.
+The Profiling tool generates information which can be used for debugging and profiling applications.
 Information such as Spark version, executor information, properties and so on. This runs on either CPU or
 GPU generated event logs.
 
-Please refer to [spark qualification tool documentation](docs/spark-qualification-tool.md)
-and [spark profiling tool documentation](docs/spark-profiling-tool.md)
```
Comment (tgravescs) [inline README.md:65]: we don't need to rename the files, lets change the names back.

File: docs/qualification-tool.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ any machine and include the jars in the classpath.
 java -cp ~/rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*:$HADOOP_CONF_DIR/ \
  com.nvidia.spark.rapids.tool.qualification.QualificationMain --help
 
-RAPIDS Accelerator for Apache Spark Qualification tool
+Qualification tool - RAPIDS Accelerator for Apache Spark
```
Comment (tgravescs) [inline docs/qualification-tool.md:86]: lets change to RAPIDS Accelerator Qualification tool for Apache Spark

File: README.md (hunk 1 above)
Comment (amahussein) [reply README.md:65]: done!

File: docs/qualification-tool.md (hunk 2 above)
Comment (amahussein) [reply docs/qualification-tool.md]: done!

**Review by tgravescs — APPROVED** [review]

**Pull request merged**